CONTRIBUTING.md

```diff
@@ -17,6 +17,15 @@ regardless of your experience level. Whether you're a seasoned OpenTelemetry
 developer, just starting your journey, or simply curious about the work we do,
 you're more than welcome to participate!
 
+Even though, anybody can contribute, there are benefits of being a member of our
+community. See to the [community membership
+document](https://github.com/open-telemetry/community/blob/main/community-membership.md)
+on how to become a
+[**Member**](https://github.com/open-telemetry/community/blob/main/community-membership.md#member),
+[**Approver**](https://github.com/open-telemetry/community/blob/main/community-membership.md#approver)
+and
+[**Maintainer**](https://github.com/open-telemetry/community/blob/main/community-membership.md#maintainer).
+
```

Comment:
While we are updating this file, can we also change the calendar link [here ](https://github.com/open-telemetry/opentelemetry-rust/blob/main/CONTRIBUTING.md#contributing-to-opentelemetry-rust)to - https://github.com/open-telemetry/community?tab=readme-ov-file#calendar

Reply:
done